docs/csharp/language-reference/keywords/checked-and-unchecked.md

```diff
@@ -11,26 +11,26 @@ helpviewer_keywords:
 ms.assetid: a84bc877-2c7f-4396-8735-1ce97c42f35e
 ---
 # Checked and Unchecked (C# Reference)
-C# statements can execute in either checked or unchecked context. In a checked context, arithmetic overflow raises an exception. In an unchecked context, arithmetic overflow is ignored and the result is truncated.  
+C# statements can execute in either checked or unchecked context. In a checked context, arithmetic overflow raises an exception. In an unchecked context, arithmetic overflow is ignored and the result is truncated by discarding any high-order bits that don't fit in the destination type.  
   
--   [checked](../../../csharp/language-reference/keywords/checked.md) Specify checked context.  
+-   [checked](checked.md) Specify checked context.  
   
--   [unchecked](../../../csharp/language-reference/keywords/unchecked.md) Specify unchecked context.  
-  
- If neither `checked` nor `unchecked` is specified, the default context depends on external factors such as compiler options.  
+-   [unchecked](unchecked.md) Specify unchecked context.  
   
  The following operations are affected by the overflow checking:  
   
 -   Expressions using the following predefined operators on integral types:  
   
-     `++` `--` - (unary)   `+` -   `*` `/`  
+     `++`, `--`, unary `-`, `+`, `-`, `*`, `/`  
   
--   Explicit numeric conversions between integral types.  
+-   Explicit numeric conversions between integral types, or from `float` or `double` to an integral type.  
   
- The [/checked](../../../csharp/language-reference/compiler-options/checked-compiler-option.md) compiler option lets you specify checked or unchecked context for all integer arithmetic statements that are not explicitly in the scope of a `checked` or `unchecked` keyword.  
+ If neither `checked` nor `unchecked` is specified, the default context for non-constant expressions (expressions that are evaluated at run time) is defined by the value of the [-checked](../compiler-options/checked-compiler-option.md) compiler option. By default the value of that option is unset and arithmetic operations are executed in an unchecked context.
+ 
+ For constant expressions (expressions that can be fully evaluated at compile time), the default overflow checking context is always checked. Unless a constant expression is explicitly placed in an unchecked context, overflows that occur during the compile-time evaluation of the expression cause compile-time errors.
```

Comment:
I think it's best to phrase this as "the default context" for consistency. Other than that, @pkulikov, this looks great.